.github/ISSUE_TEMPLATE/new_minor_release.md

```diff
@@ -0,0 +1,67 @@
+---
+name: New Minor Release
+about: Create a issue for tracking a new minor release.
+title: 'Release vX.Y.Z'
+labels: enhancement
+assignees: ''
+
+---
+
+## It is time for a new minor release for Solidus!
+
+We are committed to release every quarter in order to have a constant
+and predictable cadence for our deliverables.
+
+For more information, check out our [Release Policy](https://solidus.io/release_policy/) page.
+
+## Detailed list of steps for the release
+
+We have a pretty clear process to release a new minor version. Everything is described on our
+[Release pipeline](https://github.com/solidusio/solidus/wiki/Release-pipeline-automation) automation and [How to release Solidus](https://github.com/solidusio/solidus/wiki/How-to-release-Solidus) wiki pages,
+which we encourage to review if the assignee is not familiar with how to release Solidus.
+
+### Release preparation
+
+- Be sure there are no pending PRs that you want to include in the next minor.
+- Visit the GitHub Solidus repository.
+- Navigate to the "Actions" tab and select the "Prepare release" item.
+- Click the "Run Workflow" button, leaving `main` as the branch and the checkbox to prepare the next major unchecked (unless necesasy).
```

Comment:
```suggestion
- Click the "Run Workflow" button, leaving `main` as the branch and the checkbox to prepare the next major unchecked (unless necessary).
```

Reply:
Although, if the issue is only used for minor releases, it shouldn't b ever necessary :slightly_smiling_face:

Reply:
But the last minor still needs a minor release, isn't it? Or in that case the last minor is covered in the major release procedure?

Reply:
Oh, yeah, sorry, I got confused :slightly_smiling_face: